velocity_noise:
  x:
    mean: 0
    stddev: 0.001
  y:
    mean: 0
    stddev: 0.001
  z:
    mean: 0
    stddev: 0.001
  yaw:
    mean: 0
    stddev: 0.001